### PR TITLE
LibWeb: Implemented basic access_key_label function.

### PR DESCRIPTION
**Pull request by bokendell**
This contribution is mainly due to a group project from the University of Utah CS4560 class. 

The access_key_label function implementation was started and tested to provide the desired output. There are comments and debug lines included in the method, as we were not sure what the system modifier key was for serenity or how to get it per each OS. The same follows for determining a valid system key. The method defaults to providing "Alt+{access_key}" as the access_key_label and an empty string if not access_key is found.

Documentation followed: https://html.spec.whatwg.org/multipage/interaction.html#dom-accesskeylabel

### CODE REVIEW

File: Meta/Lagom/Contrib/VideoPlayerSDL/main.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #include <LibCore/MappedFile.h>
 #include <LibMain/Main.h>
 #include <LibVideo/PlaybackManager.h>
-#include <SDL2/SDL.h>
+#include <SDL.h>
```
Comment (MacDue) [inline Meta/Lagom/Contrib/VideoPlayerSDL/main.cpp:13]: Unrelated change.

File: Userland/Libraries/LibWeb/HTML/HTMLElement.cpp [hunk 2]
```diff
@@ -566,4 +566,20 @@ void HTMLElement::did_receive_focus()
     browsing_context->set_cursor_position(DOM::Position::create(realm(), *this, 0));
 }
 
+String HTMLElement::access_key_label() const
+{
+    auto access_key = get_attribute(HTML::AttributeNames::accesskey);
+    if (!access_key.has_value() || access_key->is_empty())
+        return {};
```
Comment (ADKaster) [inline Userland/Libraries/LibWeb/HTML/HTMLElement.cpp:573]: Add the spec steps from the algorithm in the spec above the functions as comments.

For the start of the method, something like this:

```suggestion
// Whenever an element's accesskey attribute is set, changed, or removed, the user agent must update the element's assigned access key by running the following steps:
// FIXME: We need to update the assigned access key on attribute change, set and remove.

// 1. If the element has no accesskey attribute, then skip to the fallback step below.
auto access_key = get_attribute(HTML::AttributeNames::accesskey);
if (!access_key.has_value() || access_key->is_empty()) {
    // 4. Fallback: Optionally, the user agent may assign a key combination of its choosing as the element's assigned access key and then return.
    return {};
}
```

**Review by ADKaster — COMMENTED** [review]
I'm not sure how much time y'all are trying to put into this, but if I was implementing this, I would do it the following way:

- create a member variable `Optional<String> m_assigned_access_key;`  (or even better, Web::KeyEvent or a similar type from `LibWeb/Page/InputEvent.h`. Though `Optional<String>` would be fine for now, I suspect @trflynn89 will have some ideas on how to flesh this out after this initial implementation :).

- create a member function `update_assigned_access_key()`. This member function will have the algorithm described in https://html.spec.whatwg.org/multipage/interaction.html#assigned-access-key. 

- change this function, `access_key_label` to instead "stringify" the Access key label. In the case of the member being an Optional, that's just `return m_assigned_access_key.value_or(String{});`

- add conditional statements (an else if) to HTMLElement::attribute_changed() to handle when the `"accesskey"sv` attribute is changed, and call the `update_assigned_access_key()` method.

File: Userland/Libraries/LibWeb/HTML/HTMLElement.cpp [hunk 3]
```diff
@@ -566,4 +566,20 @@ void HTMLElement::did_receive_focus()
...
+
+    StringBuilder access_key_label;
+    dbgln("FIXME: Implement HTMLElement::access_key_label() valid system key check");
+    // Loop through words in access_key to find first valid system key
+    dbgln("FIXME: Implement HTMLElement::access_key_label() get system modifier key");
+    // get system modifier key and append that instead of this line
```
Comment (ADKaster) [inline Userland/Libraries/LibWeb/HTML/HTMLElement.cpp:579]: For steps that you don't want to implement in this change set, it's preferred to do something more like this:

```
// FIXME: N. Some verbatim text from the spec
// FIXME: N+1. Some other verbatim text from the spec
```

If the step that you're marking as FIXME has substeps, you can skip them if the parent step is FIXME'd.
